TradingOnTalentURAP/degree_labelling.py

```diff
@@ -0,0 +1,103 @@
+               'michigan state university.csv',
+               'new york university.csv',
+               'penn state university.csv',
+               'purdue university.csv',
+               'texas a&m university.csv',
+               'the ohio state university.csv',
+               'the university of texas at austin.csv',
+               'universidad complutense de madrid.csv',
+               'university of california, berkeley.csv',
+               'university of florida.csv',
+               'university of illinois at urbana-champaign.csv',
+               'university of michigan.csv',
+               'university of mumbai.csv',
+               'university of phoenix.csv',
+               'university of southern california.csv',
+               'university of toronto.csv',
+               'university of washington.csv']
+
+for uni in universities:
+    df = pd.read_csv("/Users/jacquelinewood/Documents/URAP/Universities_Unlabelled/" +, header = None)
+    df = df.rename(columns={0: 'Degree', 1: '# Students'})
+    dfs[uni[:-4]] = df
+
+# Label dataframes
+
+for uni in dfs:
+    df = assign_labels(dfs[uni], direct_keywords = [
+            "machine learning",
+            "artificial intelligence",
+            "natural language processing",
+                                                   "computer vision",
+                                                   "robotics",
+                                                   "ML engineering",
+                                                   "AI systems"],
+                       direct_star_keywords = ["data science","data analytics", "big data"],
+                   exclude_keywords = ['business','system','management','technology','communications','design',
+                                                                      'database',
+                                                                      'structures',
+                                                                      "data networks",
+                                                                      "machine perception",
+                                                                      "information studies",
+                                                                      "warehousing",
+                                                                      "library",
+                                                                      "journalism","media",
+                                                                 "mgmt","sports","estate","geo",
+                    "architecture","marketing","security","health","informatics","tool","information t"],
+                   tangential_keywords = ["computer",
+                      "stati",
+                      "info","applied math","informatics"])
+    
+    dfs[uni] = df
+
+# Write to csvs
+
+for uni in dfs:
+    
+    dfs[uni].to_csv("/Users/jacquelinewood/Documents/URAP/Universities_Labelled/" + uni + ' labelled.csv')
```

Comment:
Please make this a commandline argument like source directory. So, now you would run your script approximately like so...

`python degree_labelling.py /Users/jacquelinewood/Documents/URAP/Universities_Unlabelled /Users/jacquelinewood/Documents/URAP/Universities_Labelled`